hintman.cabal

```diff
@@ -24,9 +24,17 @@ library
   hs-source-dirs:      src
   exposed-modules:     Hintman
                        Prelude
-
-  build-depends:       base-noprelude
-                     , relude
+                       Hintman.App
+                       Hintman.Server
+                       Hintman.Config
+               
```

Comment:
Maybe indent these to match https://github.com/kowainik/hintman/blob/master/hintman.cabal#L27

Reply:
@cronokirby we already have this indent level in master https://github.com/kowainik/hintman/blob/master/hintman.cabal#L26
I just followed on. If need to change I will :)